Makefile

```diff
@@ -0,0 +1,2 @@
+rbacgen:
+	go run cmd/rbacgen/main.go -specs cmd/rbacgen/timetableApi.yaml -output cmd/rbacgen/roleTmpl.yaml
```

Comment:
Add two targets to apply and rollback DB schema.
e. g. 
`dev-schema-up` and `dev-schema-down`
`migrate -source file://path/to/migrations -database postgres://localhost:5432/database up`